packages/core/docs/changelog/6089.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'Fixed incorrect method`',
+  link: 'https://github.com/vuestorefront/vue-storefront/issues/6089',
+  isBreaking: false,
+  breakingChanges: [],
+  author: 'jaydubb12',
+  linkToGitHubAccount: 'https://github.com/jaydubb12/vue-storefront'
+};
```

Comment:
This file is unnecessary since this change only affects an internal logger, and `log.warning` is not even used.

We don't include internal changes in changelogs if they don't affect end developers. This makes it easier for them to upgrade after we release a new version.

Reply:
yeah  @jaydubb12 can you remove it?